### DISCUSSION TITLE
Can aiortc be used for streaming content to MR headsets?

### DISCUSSION THREAD

**Discussion by RayShing** [aiortc/aiortc#952] [Q&A]
Hello, I would like to ask, if anyone had experience using aiortc for streaming volumetric content to AR/VR/MR headsets, such as HoloLens 2 and MagicLeap?

**Comment by arthurnni** [comment]
Here is  a  sample code  for  streaming  webcam  from aiortc  to  Meta Quest 3

https://github.com/gtk2k/Unity_aiortc_sample

It works  for  local  network   but  not  over  internet.    

I am looking  for a  solution to  include  STUN  and  Signaling server  and  Streaming  over  internet.
